src/crypto/CHIPCryptoPAL.h

```diff
@@ -222,8 +222,16 @@ bool IsBufferContentEqualConstantTime(const void * a, const void * b, size_t n);
 template <typename Sig>
 class ECPKey
 {
+protected:
```

Comment:
This breaks the public API. Please stand by while we check if this break our integration. Please wait until after holidays to merge.

Reply:
Unless you're using `ECPKey` directly (rather than a sub-class) this shouldn't really change anything, but happy to wait until you've validated this doesn't break things.

Reply:
@tcarmelveilleux 

Confirmed with Albert that this will only break tests that return a copy, but mentioned that can work around that when we roll